src/Traits/PersonTrait.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * Copyright (c) 2021 Heimrich & Hannot GmbH
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace HeimrichHannot\UtilsBundle\Traits;
+
+use Contao\StringUtil;
+
+trait PersonTrait
+{
+    /**
+     * @return array
+     *               returns empty array or array of group Models
+     */
+    public function getActiveGroups(int $userId, string $source): array
+    {
+        if (!$userId) {
+            return [];
+        }
+
+        if (null === ($userModel = $this->modelUtil->findModelInstanceByIdOrAlias($source, $userId))) {
+            return [];
+        }
```

Comment:
Was ist source ist in diesem Fall table. Warum nicht einfach findModelInstanceByPk() ? Es gibt per default in Contao keine User und/oder Mitglieder-Aliase.

Bitte nenne auch $source in $table um, da Konvention. Und dokumentiere die Felder und die Methode bitte auch.